scale/job/management/commands/scale_pre_steps.py

```diff
@@ -143,17 +147,16 @@ def _generate_input_metadata(self, job_exe):
                 remote_path = self._calculate_remote_path(job_exe)
                 scale_file.file_path = remote_path
 
-                for workspace in workspace_models:
-                    try:
-                        if not input_metadata_id:
-                            ScaleFile.objects.upload_files(workspace, [FileUpload(scale_file, local_path)])
-                            input_metadata_id = ScaleFile.objects.get(file_name=file_name).id
-                            data = job_exe.job.get_job_data()
-                            data.add_file_input('INPUT_METADATA_MANIFEST', input_metadata_id)
-                            job_exe.job.input = data.get_dict()
-                            job_exe.job.save()
-                    except:
-                        continue
+                try:
```

Comment:
Should be able to remove this entire block as uploading the metadata won't be necessary.